### ISSUE TITLE
Feature Request: Default Route Option for Item Conduit Color Channels

### ISSUE THREAD

**Issue by javacraft** [SleepyTrousers/EnderIO#1928]
First and foremost, please let me extend many, many praises for this great mod. The ability to house different conduit types in a single block is a revolutionary leap forward for creating automated systems that not only are functional but are esthetically pleasing, elegant, and compact.

One feature that would further compact designs would be the ability to have a default route for items. This would function the same as an unfiltered low priority Insert for a specific channel color, but would be applicable to all channels.

For example, item conduits can be used very effectively to extract and insert bees into apiaries/alvearies. A single item conduit is used in In/Out mode to extract bees and products and reinsert princesses and drones by enabling Self-Feed and setting a higher Insert priority. Anything not reinserted is routed to a lower priority destination of the same item channel color. To eliminate cross-breeding in a production line of multiple alvearies, each is assigned a different item channel color. This ensures that drones are not accidentally inserted into alvearies of a different species. Unfortunately, this setup requires that all products and excess drones have to be separately inserted into one or more containers to "aggregate" individual channel colors such that they can be routed to their appropriate destination.

Rather than having a couple of chests sprouting item conduits on every side, multiple chests can be used to effectively "sub-network" item channels. For example, each alveary or pairs of alvearies independently feed overflow in a chest, which are then extracted elsewhere.  Chests can be daisy-chained or extracted in parallel allowing all items to wind up at a single destination via a single channel color.

A Default Route option would eliminate the need for excess use of inventories simply for the purpose of aggregating items from different channel colors. Another possible option would be the ability to specify a set or range of channel colors for which the Insert would behave as the default route. I appreciate any consideration for adding this feature in a future build providing that it will not cause problems for your existing routing algorithm.
